lib/view/staff_welfare/your_welfare_view.dart

```diff
@@ -42,7 +42,7 @@ class YourWelfareView extends StatelessWidget {
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: <Widget>[
                   _buildSpacer(),
-                  Text('It''s okay to not be okay.', style: AppStyles.textH1),
+                  Text('It' 's okay to not be okay.', style: AppStyles.textH1),
```

Comment:
@alessandrofavero extra space?

Reply:
It should be fixed now. I think it was dartfmt.